src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/InductionNotFoundException.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction
+
+/**
+ * Thrown when a Induction does not exist for a given Prisoner.
+ */
+class InductionNotFoundException(val prisonNumber: String) :
```

Comment:
I think this exception class might be in the wrong package? I think it should probably live in the domain source tree, in the same package as the `Induction` domain class ?
I think thats what we've done with our other domain classes and their exceptions?